cosmic/src/evolv2.f

```diff
@@ -2791,12 +2816,35 @@ SUBROUTINE evolv2(kstar,mass,tb,ecc,z,tphysf,
 *
          if(kstar(j2).ge.10)then
             if(acc_lim.lt.0.d0)then
-               dm2 = MIN(dm1,dme)
+*
+* If there is wind accretion the total amount of mass change is
+* dms(j2) = dmr(j2) - dmt(j2), where dmt(j2) is the accretion
+* from the companion. We should limit to the Eddington limit minus
+* the amount of accretion that is already coming in from Winds
+*
+               dm2 = MIN(dm1,dme + dms(j2))
+*
+* If we already hit supereddington wind accretion, don't add
+* any more mass through RLO
+*
+               if(supedd.eqv..true.) dm2 = 0.d0
                if(dm2.lt.dm1) supedd = .true.
             elseif(acc_lim.ge.0.d0)then
-               dm2 = MIN(acc_lim*dm1,dme)
+*
+* If there is wind accretion the total amount of mass change is
+* dms(j2) = dmr(j2) - dmt(j2), where dmt(j2) is the accretion
+* from the companion. We should limit to the Eddington limit minus
+* the amount of accretion that is already coming in from Winds
+*
+               dm2 = MIN(acc_lim*dm1,dme + dms(j2))
```

Comment:
shouldn't this be `dme  - dms(j2)`? Or is `dms(j2)` negative?

Reply:
Since dms(j2) = dmr(j2) - dmt(j2), dms(j2) is negative for CO accretors. This especially true since dmr(j2) << dmt(j2) because the wind mass loss is 0.